regression/goto-instrument/nondet-volatile-02/test.c

```diff
@@ -0,0 +1,25 @@
+#include <assert.h>
+
+volatile int a;
```

Comment:
:no_entry_sign: These are already nondet since you never assign to them. I believe main should start with assignments to 0 for all?

Reply:
The point of the test is that a is not havoc'd

Reply:
Variables with static storage duration are initialized to zero (including volatile ones).